tests/Images/Input/Pbm/00000_00000.ppm

```diff
@@ -0,0 +1,4 @@
+P6
```

Comment:
Looks like we'll need to add the extensions for these file types to our .gitconfig and also ensure that the images are stored in GitLFS. I'll do the upstream work to allow that.